packages/components/pie-notification/test/component/pie-notification.spec.ts

```diff
@@ -196,96 +164,101 @@ test.describe('PieNotification - Component tests', () => {
         test.describe('hideIcon', () => {
             const variantsWithDefaultIcon = ['info', 'success', 'warning', 'error'];
             variants.filter((variant) => variantsWithDefaultIcon.includes(variant)).forEach((variant) => {
-                test(`should not render the default icon when the variant is ${variant} and hideIcon is true`, async ({ mount, page }) => {
+                test(`should not render the default icon when the variant is ${variant} and hideIcon is true`, async ({ page }) => {
                     // Arrange
-                    await mount(PieNotification, {
-                        props: {
-                            variant,
-                            hideIcon: true,
-                        },
-                    });
+                    const notificationPage = new BasePage(page, 'notification');
+
+                    const props: Partial<NotificationProps> = {
+                        variant,
+                        hideIcon: true,
```

Comment:
Unlike before, the default state of the component in Storybook doesn't have an icon displayed. To match the old test, you need to do the following:

```suggestion
                    const props: Partial<NotificationProps> = {
                        variant,
                        iconSlot: 'Placeholder',
                        hideIcon: true,
```